src/types/types.ts

```diff
@@ -0,0 +1,32 @@
+export interface StateType {
+    news: Array<NewsItemType>;
+    comments: Array<CommentsItemType>;
```

Comment:
Лучше сокращенный вариант типизации массивом юзать, читать легче
```suggestion
    comments: CommentsItemType[];
```